charts/catalog-api/templates/secret-deploys.yaml

```
@@ -11,3 +11,20 @@ type: Opaque
 data:
   apiDbPassword:  {{ required "A valid .Values.api.postgres.password entry required" .Values.api.postgres.password | b64enc }}
   keysServiceToken:  {{ required "A valid .Values.keys.token entry required" .Values.keys.token | b64enc }}
```

Давай тут полечим отступы)

```suggestion
  apiDbPassword: {{ required "A valid .Values.api.postgres.password entry required" .Values.api.postgres.password | b64enc }}
  keysServiceToken: {{ required "A valid .Values.keys.token entry required" .Values.keys.token | b64enc }}
```